public/app/features/alerting/unified/utils/matchers.ts

```diff
@@ -144,6 +172,29 @@ export const matcherFormatter = {
   },
 } as const;
 
+export function isPromQLStyleMatcher(input: string): boolean {
+  return input.startsWith('{') && input.endsWith('}');
```

Comment:
We are assuming that the matcher cannot be something with wrong format like:
`{ {foo="bar"}}`
is this correct?

Reply:
Those characters could exist in a matcher so counting is probably not a safe way :( I can't think of a good regular expression here that covers sufficient edge-cases and I think it's safe to say that even if `isPromQLStyleMatcher` returns false the parsing of the matchers will fail spectacularly :D